http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -545,10 +566,13 @@ private String convertToRequestParameter(URIBuilder redirectUriBuilder, String r
         for ( NameValuePair parameter: forwardedQueryParams) {
             jwtClaims.setClaim(parameter.getName(), parameter.getValue());
         }
+
         jwtClaims.setClaim(STATE, state);
         jwtClaims.setClaim(REDIRECT_URI, redirectUri);
         jwtClaims.setClaim(RESPONSE_TYPE, CODE);
         jwtClaims.setClaim(CLIENT_ID, deployment.getResourceName());
+        String cryptoValue = Oidc.getCryptographicValue(sessionRandomValue);
+        jwtClaims.setClaim(NONCE, String.valueOf(cryptoValue));
```

Comment:
`String.valueOf` can be removed here

Reply:
Just thinking, could we pass the crypto value directly to the methods instead of the session random value?

Reply:
done